Index: WildPay/Repositories/ExpenditureRepository.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>using Microsoft.EntityFrameworkCore;\nusing WildPay.Data;\nusing WildPay.Interfaces;\nusing WildPay.Models.Entities;\n\nnamespace WildPay.Repositories\n{\n    public class ExpenditureRepository : IExpenditureRepository\n    {\n        private readonly WildPayDbContext _context;\n\n        public ExpenditureRepository(WildPayDbContext context)\n        {\n            _context = context;\n        }\n        \n        public async Task<AddExpenditureAsync>()\n\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/WildPay/Repositories/ExpenditureRepository.cs b/WildPay/Repositories/ExpenditureRepository.cs
--- a/WildPay/Repositories/ExpenditureRepository.cs	
+++ b/WildPay/Repositories/ExpenditureRepository.cs	
@@ -14,7 +14,7 @@
             _context = context;
         }
         
-        public async Task<AddExpenditureAsync>()
+
 
     }
 }
